From fc60aba05c1686d31ecc82ef1fe478347c688a39 Mon Sep 17 00:00:00 2001
From: Shizhao Liu <lshizhao@amazon.com>
Date: Mon, 9 Feb 2026 11:27:01 -0800
Subject: [PATCH 18/18] Change ova partition type back to ext4

Temporarily revert the change of ova partition type
back to ext4.

Signed-off-by: Shizhao Liu <lshizhao@amazon.com>
---
 .../capi/packer/ova/linux/ubuntu/http/22.04.efi/user-data.tmpl  | 2 +-
 images/capi/packer/ova/linux/ubuntu/http/22.04/user-data.tmpl   | 2 +-
 .../capi/packer/ova/linux/ubuntu/http/24.04.efi/user-data.tmpl  | 2 +-
 images/capi/packer/ova/linux/ubuntu/http/24.04/user-data.tmpl   | 2 +-
 4 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/images/capi/packer/ova/linux/ubuntu/http/22.04.efi/user-data.tmpl b/images/capi/packer/ova/linux/ubuntu/http/22.04.efi/user-data.tmpl
index 8ee383fc9..a6c61f463 100644
--- a/images/capi/packer/ova/linux/ubuntu/http/22.04.efi/user-data.tmpl
+++ b/images/capi/packer/ova/linux/ubuntu/http/22.04.efi/user-data.tmpl
@@ -68,7 +68,7 @@ autoinstall:
       grub_device: false
       type: partition
       id: partition-1
-    - fstype: xfs
+    - fstype: ext4
       volume: partition-1
       preserve: false
       type: format
diff --git a/images/capi/packer/ova/linux/ubuntu/http/22.04/user-data.tmpl b/images/capi/packer/ova/linux/ubuntu/http/22.04/user-data.tmpl
index 63331fb21..de47053e8 100644
--- a/images/capi/packer/ova/linux/ubuntu/http/22.04/user-data.tmpl
+++ b/images/capi/packer/ova/linux/ubuntu/http/22.04/user-data.tmpl
@@ -54,7 +54,7 @@ autoinstall:
       - type: format
         id: format-0
         volume: partition-0
-        fstype: xfs
+        fstype: ext4
         preserve: false
       - type: mount
         id: mount-0
diff --git a/images/capi/packer/ova/linux/ubuntu/http/24.04.efi/user-data.tmpl b/images/capi/packer/ova/linux/ubuntu/http/24.04.efi/user-data.tmpl
index 37b67cf07..b579ea069 100644
--- a/images/capi/packer/ova/linux/ubuntu/http/24.04.efi/user-data.tmpl
+++ b/images/capi/packer/ova/linux/ubuntu/http/24.04.efi/user-data.tmpl
@@ -68,7 +68,7 @@ autoinstall:
       grub_device: false
       type: partition
       id: partition-1
-    - fstype: xfs
+    - fstype: ext4
       volume: partition-1
       preserve: false
       type: format
diff --git a/images/capi/packer/ova/linux/ubuntu/http/24.04/user-data.tmpl b/images/capi/packer/ova/linux/ubuntu/http/24.04/user-data.tmpl
index 3e91ea071..5582f2df5 100644
--- a/images/capi/packer/ova/linux/ubuntu/http/24.04/user-data.tmpl
+++ b/images/capi/packer/ova/linux/ubuntu/http/24.04/user-data.tmpl
@@ -54,7 +54,7 @@ autoinstall:
       - type: format
         id: format-0
         volume: partition-0
-        fstype: xfs
+        fstype: ext4
         preserve: false
       - type: mount
         id: mount-0
-- 
2.50.1 (Apple Git-155)

